easybuild/easyconfigs/c/CGAL/CGAL-4.14-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,38 @@
+name = 'CGAL'
+version = '4.14'
```

Comment:
@furstj A 4.14.1 was released just yesterday, let's bump to that?

See https://github.com/CGAL/cgal/releases/tag/releases%2FCGAL-4.14.1